### DISCUSSION TITLE
On DEBUG mode my functions is called twice instead of once

### DISCUSSION THREAD

**Discussion by ghost** [pallets/flask#5012] [Q&A] [answered]

**Comment by ThiefMaster** [comment]
How do you expect any help without posting how you run your app nor any code (ideally a minimal example)?

But a wild guess would be that you run it at import time and it runs once and then again when the reloader restarts the app within the reloader. In any case, this is something for discussions and not an issue.

**Comment by davidism** [comment, marked as answer]
Remember to search first. I literally searched for "flask twice" and got this as the first result: https://stackoverflow.com/questions/25504149/why-does-running-the-flask-dev-server-run-itself-twice

**Reply by ghost** [reply]
oh sorry :( i am new in github
